### PR TITLE
[pt-br] Add docs/tasks/tools/_index

### PR DESCRIPTION
**Pull request by MrErlison**
Add translation to Brazilian Portuguese:
- pt-br/docs/tasks/tools/_index.md

/cc @stormqueen1990 @devlware @edsoncelio

### CODE REVIEW

File: content/pt-br/docs/tasks/tools/_index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+## kubectl
+
+<!-- overview -->
+A ferramenta de linha de comando do Kubernetes, [kubectl](/docs/reference/kubectl/kubectl/), permite que você execute comandos nos clusters Kubernetes. 
+Você pode usar o kubectl para instalar aplicações, inspecionar e gerenciar recursos de cluster e visualizar os logs. 
+Para obter mais informações, incluindo uma lista completa de operações kubectl, consulte a [documentação de referência `kubectl`](/docs/reference/kubectl/).
+
+kubectl is installable on a variety of Linux platforms, macOS and Windows.
```
Comment (edsoncelio) [inline content/pt-br/docs/tasks/tools/_index.md:15]: untranslated sentence.

**Review by devlware — COMMENTED** [review]
/lgtm

**Comment by stormqueen1990** [comment]
Thanks @MrErlison! This is looking good but it appears the branch needs a rebase, as the build is currently failing.

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**